rpc/rpc.go

```diff
@@ -1395,12 +1382,12 @@ func (c *Conn) handleDisembargo(ctx context.Context, d rpccp.Disembargo, release
 			}
 
 			iface := ptr.Interface()
-			if !iface.IsValid() || int64(iface.Capability()) >= int64(len(ans.resultCapTable)) {
+			if !iface.IsValid() || int64(iface.Capability()) >= int64(len(ans.results.Message().CapTable)) {
 				err = rpcerr.Failedf("incoming disembargo: sender loopback requested on a capability that is not an import")
 				return
 			}
 
-			client := ans.resultCapTable[iface.Capability()] //.AddRef()
+			client := iface.Client() //.AddRef()
```

Comment:
Do we want to keep the `// .AddRef()` comment?

Reply:
No need I think; removed.